guidelines/1.3.2.md

```diff
@@ -1,46 +1,94 @@
 
-## What?
+In most cases, a 'logical reading order' is top-to-bottom and left-to-right. 
 
-When the order of content is important, the content order must be preserved no matter how it is presented to the user.
+---
+
+## Requirements
 
-### Requirements
+#### Requirements for both native and web
 
-* The visual order of content matches the underlying code order;
-* Tabular data is presented using proper HTML markup (`<table>`, `<tr>`, `<th>`, and `<td>` elements).
+* When users navigate the page using a keyboard's 'tab' key, or cursor arrows (when reader is on), elements receive the [keyboard focus](./definitions.md#keyboard-focus) and [screen reader cursor]() following a predictable, natural reading order.
+  * A natural reading order is from left to right and top to bottom in most cases in Western countries.
+* The easiest way to avoid focus order issues is to make sure that elements are displayed on screen in the same order as they appear in the DOM (Web) / Outline View (iOS and Android).
 
-### Official guidelines
+#### Requirements for both native and web
 
-* [1.3.2 "Meaningful sequence" (A) in the Web Content Accessibility Guidelines](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html)
+* Tabular data is presented using proper HTML markup (`<table>`, `<tr>`, `<th>`, and `<td>` elements)
 
-## Why?
+### Why?
 
 This ensures that the order of content is meaningful whether someone is looking at it, listening to it with a screen reader, or has stripped out visual styling using a browser plugin.
 
-## How?
+### Official wording in the Web Content Accessibility Guidelines
+
+> [**1.3.2 Meaningful Sequence:**](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html) When the sequence in which content is presented affects its meaning, a [correct reading sequence](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html#correct-reading-sequencedef) can be [programmatically determined](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html#programmaticallydetermineddef). (Level A)
+
+See the [W3C's detailed explanation of this guideline](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html) with techniques and examples.
+
+---
+
+## Guidance for Design
 
-### iOS
+* When you provide different mock-ups for different screen sizes (e.g. desktop and mobile, or for different break points), make sure that elements appear in the same order across on the different mock-ups.
+  * If you can't do that, call it out to developers as they're likely to need to create a element structure for the different screen sizes
+* On the mock-ups that you give to developers or in conversations, indicate what the expected content order is (especially for interactive elements).
 
-Setting a custom order for child elements:
+### Common mistakes
+
+* Imagine that you have a dialog has two buttons: 'Cancel' and 'OK'.
```

Comment:
`Imagine that you have a dialog has two buttons` doesn't seem quite right to me?

Reply:
Thanks. I'll try to improve